src/reducers/view.js

```diff
@@ -100,6 +105,8 @@ const files = (state = [], action) => {
       const clone = state.slice(0)
       action.files.forEach((f, i) => { clone[action.skip + i] = f })
       return clone
+    case RENAME_SUCCESS:
+      return updateItem(action.file, state)
```

Comment:
`view.js` is very big ;) it's reducers of files. We will have to refactoring a lot.